inst/stan/wwinference.stan

```diff
@@ -127,9 +127,9 @@ parameters {
   real<lower=0> sigma_growth;
   real<lower=-1, upper=1> initial_growth; // initial growth from I0 to first observed time
   real<lower=1/sqrt(5000)> inv_sqrt_phi_h;
-  real<lower=0> sigma_ww_site_mean; //mean of site level stdev
-  real<lower=0> sigma_ww_site_sd; // stdev of site level stdev
-  vector<lower=0>[n_ww_lab_sites]sigma_ww_site_raw; // let each lab-site combo have its own observation error
+  real mode_sigma_ww_site; //mode of site level stdev
+  real<lower=0> sd_log_sigma_ww_site; // stdev of the log site level stdev
```

Comment:
Isn't this now a more wider prior because the prior values of the stdev are the same, but this is now in log scale?